platform/vendor/vendor.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -euo pipefail
+
+go install github.com/trmiller/vendorme@latest
```

Comment:
Not to be a stickler but can you also pin the version of vendorme itself to a commit? Once you have a "release" we should probably look at having cosign sign the releases